src/openforms/plugins/plugin.py

```diff
@@ -43,7 +46,7 @@ def check_config(self):
         """
         raise NotImplementedError()
 
-    def get_config_actions(self) -> list[tuple[str, str]]:
+    def get_config_actions(self) -> Sequence[Action]:
```

Comment:
Altough I usually like to use the most precise type for return annotations, it makes sense in this case as the way to say "I only guarantee the `Sequence` protocol to be respected" :)

Reply:
there's actually a warning about covariance here if you *don't* include the `list[Action]` type annotation, and for how we use it, list/tuple/sequence doesn't make any difference so this was the easiest fix